tests/test_generate_agents_common.py

```
@@ -0,0 +1,426 @@
+
+from __future__ import annotations
+
+import os
+import sys
+from pathlib import Path
+
+# Add build directory to path for imports
+_BUILD_DIR = Path(__file__).resolve().parent.parent / "build"
+sys.path.insert(0, str(_BUILD_DIR))
+
+from generate_agents_common import (  # noqa: E402
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)